packages/data/src/api/nearPrice/nearPrice.ts

```diff
@@ -0,0 +1,41 @@
+import { BINANCE_API, COIN_GECKO_API } from '../../constants';
+import { BinanceNearPriceData, CoinGeckoNearPriceData, NearPriceData, NearPriceError } from './nearPrice.types';
+
+
+export const nearPriceFallback = async (): Promise<CoinGeckoNearPriceData> => {
+  const geckoApi = await fetch(
+    COIN_GECKO_API,
+  );
+  const data = geckoApi.json();
+  return data;
+};
+
+
+const nearPriceData = async (): Promise<NearPriceData | NearPriceError> => {
+
+  try {
+    const req = await fetch(
+      BINANCE_API,
+    );
+
+    const res: BinanceNearPriceData = await req.json();
+
+    if (!res.price) {
+      const price = await nearPriceFallback();
+      return { nearPrice: price.near.usd };
+    }
+
+    return { nearPrice: res.price };
```

Comment:
if you return as it is now on the nearPrice function you will need to return price.nearPrice
```suggestion
    return res.price ;
```